opm/parser/eclipse/EclipseState/Eclipse3DProperties.cpp

```diff
@@ -854,7 +853,17 @@ namespace Opm {
         }
     }
 
-    // static member function
+    
+    const MessageContainer& Eclipse3DProperties::getMessageContainer() const {
```

Comment:
If the question is whether `getIntGridProperty` and `getDoubleGridProperty` should take `messageContainer` as a parameter, I think the clear answer is _no_.  If you want to add a new input argument to `initPORV` you also have to add a new input argument to _every postprocessor and initializer_.  I don't think that is a good idea.